File: codegen/codegen-erlang/src/test/resources/ir/client_dispatch_get_name_retry.expected.erl
RetryOpts = maps:get(retry, Config, #{}),
retry_mod:with_retry(
    fun() ->
        Req = http_service_rest_json_1:encode_get_name_request(Input),
        case runtime_http:dispatch(Config, Req) of
            {ok, Resp} ->
                http_service_rest_json_1:decode_get_name_response(Resp);
            {error, Reason} ->
                {error, Reason}
        end
    end,
    RetryOpts
).

File: codegen/codegen-elixir/src/test/resources/ir/client_dispatch_get_name_retry.expected.ex
retry_opts = Map.get(config, :retry, [])
RuntimeHttp.with_retry(
  fn ->
    req = HttpServiceRestJson1.encode_get_name_request(input)
    case RuntimeHttp.dispatch(config, req) do
      {:ok, resp} -> HttpServiceRestJson1.decode_get_name_response(resp)
      {:error, reason} -> {:error, reason}
    end
  end,
  Keyword.merge([{:should_retry, &HttpServiceClient.should_retry?/1}], retry_opts)
)
